src/sentry/api/endpoints/organization_releases.py

```diff
@@ -564,9 +565,18 @@ def post(self, request: Request, organization) -> Response:
                     project_ids=[project.id for project in projects],
                     user_agent=request.META.get("HTTP_USER_AGENT", ""),
                     created_status=status,
-                    auth_type="api_token" if is_api_token_auth(request.auth) else None,
+                    auth_type="api_token"
+                    if is_api_token_auth(request.auth)
+                    else "org_auth_token"
+                    if is_org_auth_token_auth(request.auth)
+                    else None,
```

Comment:
This shows up a few times. Should we have a function that can return the `auth_type` for `request.auth`?

Reply:
true, makes sense 👍 will extract this out!